thepeer-android/src/main/java/co/thepeer/sdk/utils/WebInterface.kt

```diff
@@ -101,15 +101,15 @@ internal class WebInterface(private val activity: FragmentActivity) {
     fun sendResponse(response: String) {
         Logger.log(this, response)
         val event: ThePeerEvent? = convertToGsonFromString(response)
-        when (event?.event?.getFirstPart()) {
+        when (event?.data?.channel) {
```

Comment:
Hey man @Nsikaktopdown , I noticed this change.

What was this for? 
I believe the APK I test ran with the former code?

Reply:
I was using the event value which looked like this `checkout.success`. So I used the `getFirstPart()` to get the first part which will be `checkout`. I later found the channel key which better serves the purpose. The whole logic routes the side effect based on the triggered channel e.g `send` or `checkout`